<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-PSK9T3J" failures="1" tests="1" name="Default test" time="48.83" errors="0" timestamp="2023-11-28T17:25:18 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="38.631">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at stepdefinitions.TawaraqCommodityPurchaseMakerApplicationDetails.user_validate_the_classification_value_displayed_correctly_under_application_details_screen(TawaraqCommodityPurchaseMakerApplicationDetails.java:164)
at âœ½.User_608 validate the Classification value displayed correctly under Application details screen(file:///C:/Users/inindc00604/git/ULS_Bharani/ULS_Bharani/src/test/java/features/TawaraqCommodityPurchaseMakerApplicationDetails.feature:14)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
